react/features/dynamic-branding/reducer.ts

```diff
@@ -156,6 +156,7 @@ export interface IDynamicBrandingState {
     logoImageUrl: string;
     muiBrandedTheme?: boolean;
     premeetingBackground: string;
+    showGiphyIntegration?: boolean;
```

Comment:
why not go with the negation? hideGiphyIntegration. Since I see that we want the default as show

Reply:
Yes, well that's how it's called in the server response so...